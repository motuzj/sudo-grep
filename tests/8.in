search two words in multiple files
"a erat" tests/texts/text1.txt tests/texts/text2.txt tests/texts/text3.txt tests/texts/text4.txt
