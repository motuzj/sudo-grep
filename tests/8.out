Non consectetur a erat nam at lectus urna.
elit non euismoda erat
Erosa erat nam porta rutrum
Amatomaa erat nam borrar el cer
Chtenff a erat r'luh ngnw Azathothnth
